test/pg-cdc/statistics.td

```diff
@@ -137,17 +136,16 @@ true
   ORDER BY s.name
 mz_source 1 1
 
-# TODO: database-issues#8511 (introspection tables)
-# # Ensure subsource stats show up, and then are removed when we drop subsources.
-# > SELECT
-#     t.name,
-#     SUM(u.updates_committed) > 0
-#   FROM mz_sources t
-#   JOIN mz_internal.mz_source_statistics_raw u ON t.id = u.id
-#   WHERE t.name IN ('alt')
-#   GROUP BY t.name
-#   ORDER BY t.name
-# alt true
+# Ensure subsource stats show up, and then are removed when we drop subsources.
+> SELECT
+    t.name,
+    SUM(u.updates_committed) > 0
+  FROM mz_sources t
```

Comment:
@def- it was a test issue - this needs to be `FROM mz_tables t` since source tables don't show up in `mz_source`, only `mz_tables`

Reply:
Perfect, works for me locally.